CONTRIBUTING.md

```diff
@@ -104,41 +102,55 @@ variables:
 
 `.\script\configure-integration-tests.ps1`
 
-Once these are set, the integration tests will be executed both when 
-running the IntegrationTests build target, or when running the 
-Octokit.Tests.Integration assembly in the Visual Studio test runner.
+After running this, ensure any existing instances of Visual Studio are restarted
+so they pick up the new environment variables are detected.
+
+With these variables set, you can run the integration tests locally using
+`.\build IntegrationTests` or by running the `Octokit.Tests.Integration`
+assembly in the Visual Studio test runner.
+
+**Note:** as the integration tests rely on using the actual GitHub API, you may
+encounter issues if running the tests too frequently. Please use a test account
+so that the unlikely scenario of your account being identified as a spammer.
+
+### Testing Documentation
+
+If you are making changes to the documentation for Octokit, you can test these
+changes locally using the [guide](https://github.com/shiftkey/octokit.net/blob/rewrite-contributing/docs/contributing.md)
+under the `docs` folder.
 
 ### Submitting Changes
 
-You can publish your branch from GitHub for Windows, or run this command from
+You can publish your branch from GitHub Desktop, or run this command from
 the Git Shell:
 
 `git push origin MY-BRANCH-NAME`
 
 Once your changes are ready to be reviewed, publish the branch to GitHub and
-[open a pull request](https://help.github.com/articles/using-pull-requests) 
+[open a pull request](https://help.github.com/articles/using-pull-requests)
 against it.
 
 A few little tips with pull requests:
 
  - prefix the title with `[WIP]` to indicate this is a work-in-progress. It's
    always good to get feedback early, so don't be afraid to open the PR before it's "done".
- - use [checklists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) 
+ - use [checklists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
    to indicate the tasks which need to be done, so everyone knows how close you are to done.
  - add comments to the PR about things that are unclear or you would like suggestions on
 
-Don't forget to mention in the pull request description which issue/issues are 
+Don't forget to mention in the pull request description which issue/issues are
```

Comment:
Might be good to explain how to mention the issue in the PR, since there's a few cases of people saying "fixed issue nnn" rather than "fixed issue #nnn").  I tried to find a good/official link but unfortunately there is nothing concise... there is a 5 year old ["introducing issue mentions" blog post](https://github.com/blog/957-introducing-issue-mentions) (too old) the [closing issues via commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) (more about commit messages than PR text), and things like [mastering issues](https://guides.github.com/features/issues/) are perhaps too much detail.  

So maybe just a few more words here like 
> which issue/issues are being addressed**, using the #1145 notation**